setup.py

```diff
@@ -37,7 +37,7 @@ def get_packages(package):
     author='Tom Christie',
     author_email='tom@tomchristie.com',
     packages=get_packages('mkautodoc'),
-    install_requires=[],
+    install_requires=["Markdown>=3.4.1"],
```

Comment:
Please don't pin the version. It is unnecessary, and would create a conflict with MkDocs, which currently has `<3.4`

Reply:
```suggestion
    install_requires=["Markdown"],
```